modules/test/playwright/tests/document-library-web/fileEntry.spec.ts

```diff
@@ -0,0 +1,52 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import {expect, mergeTests} from '@playwright/test';
+import {format} from 'date-fns';
+
+import {documentLibraryPagesTest} from '../../fixtures/documentLibraryPages.fixtures';
+import {featureFlagsTest} from '../../fixtures/featureFlagsTest';
+import {loginTest} from '../../fixtures/loginTest';
+import getRandomString from '../../utils/getRandomString';
+
+export const test = mergeTests(
+	loginTest(),
+	featureFlagsTest({
+		'LPD-10701': true,
+	}),
+	documentLibraryPagesTest
+);
+
+test('LPD-16658 Show a success message after scheduling a new file', async ({
+	documentLibraryEditFilePage,
+	documentLibraryPage,
+	page,
+}) => {
+	const scheduleDate = `01/01/${new Date().getFullYear() + 1}`;
+	const title = getRandomString();
+
+	const fileEntry = page.getByRole('link', {name: title});
```

Comment:
Inline this in the expect sentence

Reply:
ok, I will push an additional commit to inline this and rename the test suite to testFeatureFlagsEnabled